packages/contracts/deploy/100_install_on_management_dao/100_install.ts

```diff
@@ -0,0 +1,276 @@
+import {VERSION, METADATA} from '../../plugin-settings';
+import {
+  getManagementDao,
+  isValidAddress,
+  getLatestContractAddress,
+  findPluginRepo,
+  isPermissionSetCorrectly,
+} from '../../utils/helpers';
+import {
+  getNamedTypesFromMetadata,
+  findEventTopicLog,
+} from '@aragon/osx-commons-sdk';
+import {
+  DAO_PERMISSIONS,
+  Operation,
+  PLUGIN_SETUP_PROCESSOR_PERMISSIONS,
+} from '@aragon/osx-commons-sdk';
+import {
+  PluginSetupProcessor__factory,
+  PluginSetupProcessorEvents,
+  DAOStructs,
+} from '@aragon/osx-ethers';
+import {defaultAbiCoder, keccak256} from 'ethers/lib/utils';
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+
+/**
+ * Prints information about the used/forked network and initial deployer wallet balance.
+ * @param {HardhatRuntimeEnvironment} hre
+ */
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {ethers} = hre;
+
+  const [deployer] = await hre.ethers.getSigners();
+
+  const approvers = process.env.MANAGEMENT_DAO_MULTISIG_APPROVERS?.split(
+    ','
+  ) || [deployer.address];
+  const minApprovals = parseInt(
+    process.env.MANAGEMENT_DAO_MULTISIG_MIN_APPROVALS || '1'
+  );
+
+  const listedOnly = process.env.MANAGEMENT_DAO_MULTISIG_LISTED_ONLY;
```

Comment:
Setting the variable to `"false"` would be treated as a truthy value, we should read it as a string.

```suggestion
  const listedOnly = ["", "false"].includes(process.env.MANAGEMENT_DAO_MULTISIG_LISTED_ONLY || "");
```